src/screens/LoginView/vuelos.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
+	"folders": [
+		{
+			"path": "C:\\Users\\Equipo1\\Desktop\\BrightCoders\\kata-05-reservacion-de-vuelos-aug21-k05-t01"
```

Comment:
I dont think this information is necessary to upload to GitHub.

Reply:
I hadn´t seen it, already deleted.